dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/utils/WorkManagerUtils.kt

```diff
@@ -18,6 +18,10 @@ import java.util.concurrent.TimeUnit
 
 internal const val CANCEL_ERROR_MESSAGE = "Error cancelling the UploadWorker"
 internal const val SETUP_ERROR_MESSAGE = "Error while trying to setup the UploadWorker"
+internal const val UPLOAD_WORKER_WAS_CANCELLED_MESSAGE =
+    "A new foreground process started. UploadWorker was canceled."
+internal const val UPLOAD_WORKER_WAS_TRIGGERED_SCHEDULED =
```

Comment:
probably should be `UPLOAD_WORKER_WAS_SCHEDULED_MESSAGE`?

Reply:
yes...I know for sure I changed that but not sure what happened :)